runners/direct-java/build.gradle

```diff
@@ -72,17 +73,12 @@ dependencies {
   shadow library.java.vendored_grpc_1_26_0
   shadow library.java.joda_time
   shadow library.java.slf4j_api
-  shadow library.java.args4j
+  compile library.java.jackson_databind
```

Comment:
Is a `compile` time dependency required here, or can this be shadowed?

Reply:
shadowed jackson_databind.